wallets/rn_cli_wallet/src/navigators/ConnectionsStack.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+
+import {useTheme} from '@/hooks/useTheme';
+import ConnectionsScreen from '@/screens/ConnectionsScreen';
+import {ConnectionsStackParamList} from '@/utils/TypesUtil';
+
+const Stack = createNativeStackNavigator<ConnectionsStackParamList>();
+
+export default function ConnectionsStack() {
+  const Theme = useTheme();
+
+  return (
+    <Stack.Navigator
+      screenOptions={{
+        contentStyle: {backgroundColor: Theme['inverse-100']},
+        headerStyle: {backgroundColor: Theme['inverse-100']},
+        headerTitleStyle: {
+          color: Theme['fg-100'],
+        },
+      }}>
+      <Stack.Screen
+        name="ConnectionsScreen"
+        options={{headerTitle: 'Connections', headerLargeTitle: true}}
+        component={ConnectionsScreen}
+      />
+    </Stack.Navigator>
```

Comment:
do we need a stack for connections and settings?

Reply:
The `headerLargetTitle` of React Navigation requires you to use Stack Navigator, we cannot use this prop with Tab Navigator. In this case, I needed to move the ConnectionsScreen into it's own StackNavigator and use `headerLargetTitle`.

See: https://reactnavigation.org/docs/native-stack-navigator/#headerlargetitle

Reply:
aahh okey! Seems a little bit of an overkill to me having 3 stacks for the title effect in iOS 🤔 the title could be fixed at top and then the flatlist below, as in the iOS wallet sample

Reply:
<img width="454" alt="Screenshot 2024-02-26 at 16 40 02" src="https://github.com/WalletConnect/react-native-examples/assets/25931366/593a3a7e-2407-4cac-b158-c9cc5b93bd74">

can you reduce that extra padding at top?

Reply:
@ignaciosantise you mean between the status bar and header title? That gap is from iOS defaults, not manually set and I'm not sure if we should/could do it. Checkout WhatsApp iOS as an example:

<img width="300" src="https://github.com/WalletConnect/react-native-examples/assets/19428358/9c2bc5dd-2845-4e6e-b655-7c784f55b31a"/>

I followed the Swift sample wallet since it has the same header type:

<img width="300" src="https://github.com/WalletConnect/react-native-examples/assets/19428358/1f6d17a4-da3a-4743-a237-08eae777a97d"/>

On Android, since it doesn't have a large header feature, it'll be seen like a regular header:

<img width="300" alt="Screenshot 2024-02-27 at 10 23 53" src="https://github.com/WalletConnect/react-native-examples/assets/19428358/101c6d14-3b01-4cd1-893c-600222b873d0">